hooks/tk-multi-publish2/maya/publish_session_model.py

```diff
@@ -59,16 +61,45 @@ def validate(self, task_settings, item):
         """
         # create list of elements from maya nodes to collect results about
         workflow_data = {"elements": []}
+        # TODO: replace with find_model_root_nodes from modelpublish-8.7.0+
         toplevel_objects = findModelRootNodes()
+        valid_node_name = item.context.entity['name'].lower()
```

Comment:
Waiting to determine whether we should use `entity['name']` or `task['name']` depending on the department's workflow (creating subassets called x or creating task x within the asset itself)